run_test.sh

```diff
@@ -1,8 +1,42 @@
 # Script run on Travis CI
 # Runs the Jetpack extension tests
 
-# Set the firefox binary path
-export JPM_FIREFOX_BINARY=$TRAVIS_BUILD_DIR/../firefox/firefox
+# Check for dependencies
+if ! [ -f node_modules/istanbul/lib/cli.js ]; then
+  echo "You need to have 'istanbul' installed to run the tests."
+  echo "Run 'npm install istanbul'"
+  exit 1
+fi
+
+# Start node server
+node server.js &
+
+if [ -z "$TRAVIS" ]; then
+  # Local Env
+  echo "Local Environment"
+  export JPM_FIREFOX_BINARY=`which firefox`
+else
+  # Travis CI
+  echo "Travis CI"
+  export JPM_FIREFOX_BINARY=$TRAVIS_BUILD_DIR/../firefox/firefox
+fi
+
+current_dir=`pwd`
+current_dir+="/"
+sed -i "s:replace_with_path:$current_dir:g" test.json
```

Comment:
Since the approach of replacing a magic string is somewhat brittle it's probably a good idea to add a `git checkout test.json` in case someone decides to replace the string on their own.